creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -92,15 +90,20 @@ const healthCheckVerboseController = async (req) => {
 }
 
 /**
- * Controller for `health_check/fileupload` route
- * Calls `healthCheckFileUploadService`.
+ * Controller for `health_check/fileupload` route *
+ * Perform a file upload health check limited to configured delegateOwnerWallet.
+ * This prunes the disc artifacts created by the process after.
  */
 const healthCheckFileUploadController = async (req) => {
   const err = req.fileFilterError || req.fileSizeError
+  try {
+    await removeTrackFolder(req, req.fileDir)
+  } catch (e) {
+    throw new Error(e)
```

Comment:
just looked at `removeTrackFolder` and it already has a try-catch and never throws an error. so this try-catch block would never get hit. simplest change would be to have removeTrackFolder return error else null on success and in this calling block throw if resp is set

Reply:
I adjusted the error flow for this, but I'm still not sure if my new way to do it is the best way we could handle the error returned from `removeTrackFolder`. Please let me know what you think when you have time, thanks!